CMakeLists.txt

```diff
@@ -346,6 +346,7 @@ endif()
 install(
   TARGETS uhdm uhdm-cmp uhdm-dump uhdm-hier
   ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/uhdm
+  LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/uhdm
```

Comment:
While at it, in Surelog, we use `${CMAKE_INSTALL_LIBDIR}/uhdm` (which then ends up installing in lib64 on some systems). Is that the more canonical form we should use here as well ?

Reply:
yeah we should probably make these the same